File: src/main/java/br/com/zup/academy/dominio/repository/PaisRepository.java
package br.com.zup.academy.dominio.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import br.com.zup.academy.dominio.modelo.Pais;

@Repository
public interface PaisRepository extends JpaRepository<Pais, Long>{

}

File: casa-do-codigo-grpc/src/main/kotlin/br/com/zup/dominio/repository/PaisRepository.kt
package br.com.zup.dominio.repository

import br.com.zup.dominio.modelo.Pais
import io.micronaut.data.annotation.Repository
import io.micronaut.data.jpa.repository.JpaRepository

@Repository
interface PaisRepository: JpaRepository<Pais, Long> {

}
